tests/base/src/test/java/org/wildfly/security/http/HttpAuthenticatorTest.java

```diff
@@ -255,5 +256,24 @@ public void testUsingSecurityProviderServerMechanismWithDigestMD5() throws Excep
 
         authenticateWithDigestMD5();
     }
-
+    @Test
+    public void testBearerAuthenticationMechanism() throws Exception{
+        HttpServerAuthenticationMechanism mechanism = bearerFactory.createAuthenticationMechanism(BEARER_TOKEN,Collections.emptyMap(),getCallbackHandler("remoteUser","testrealm@host.com",null,"random"));
```

Comment:
Just a minor, we can use null instead of the "remoteUser" since we do not need a username when using bearer mechanism